packages/lit/src/test/node-imports.ts

```diff
@@ -45,3 +45,7 @@ import 'lit/html.js';
 import 'lit/experimental-hydrate-support.js';
 import 'lit/experimental-hydrate.js';
 import 'lit/static-html.js';
+
+import assert from 'node:assert/strict';
+import {isServer} from 'lit';
+assert.ok(isServer, 'Expected isServer to be truthy');
```

Comment:
Same comment as above, suggest `assert.equal`

Reply:
Also used `assert.strictEqual` although it appears that importing from `node:assert/strict` makes everything strict, so `assert.equal` will behave as `assert.strictEqual`.